src/cargo/util/config/mod.rs

```diff
@@ -228,9 +228,11 @@ impl Config {
                 .into_path_unlocked()
         });
         let wrapper = self.maybe_get_tool("rustc_wrapper")?;
+        let workspace_path = self.maybe_get_tool("workspace_rustc")?;
```

Comment:
I would maybe call this `"rustc_workspace_wrapper"`, to make it similar to the existing env var, and to make it clear that it is a wrapper (first arg is the real rustc).

Reply:
I'm still unclear why we want it to occupy the wrapper location rather than letting people set an alternative rustc. clippy-driver works fine as a wrapper or as rustc itself.

Reply:
In this case, it's probably not significant. The original motivation was https://github.com/rust-lang/rust-clippy/pull/2087 so that `RUSTC` would still work for build scripts.  I just think the wrapper variant is more flexible, since it gives the wrapper the choice of either executing the real rustc with the correct path, or replacing it (like clippy does).